logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/conversation/CreateGroupConversationUseCaseTest.kt

```diff
@@ -137,6 +149,40 @@ class CreateGroupConversationUseCaseTest {
             .wasInvoked(exactly = once)
     }
 
+    @Test
+    fun givenNameMembersAndOptions_whenCreatingGroupConversation_thenPersistSystemMessageForReceiptMode() = runTest {
```

Comment:
Could you also test for when the user is not a team member, so it doesn't insert the system message?